src/Compiler/NodeVisitor/Behavior/ComponentsNodeVisitorBehavior.php

```diff
@@ -33,6 +35,16 @@ private function findComponentByName(array $components, string $componentName):
             if (count($componentNameParts) === 0) {
                 return $component;
             }
+            $componentType = $component->getType();
+            if ($componentType instanceof ObjectType && $componentType->isInstanceOf('Nette\Application\UI\Multiplier')->yes()) {
```

Comment:
I don’t like that Multiplier is mentioned here in node visitor, it should be generic here. Any other component can beháva like multiplier in a wild. Multiplier components should be just collected in component collector (special one for multiplier).

Reply:
Not sure how it would work. How would we pass info that it is multiplier-like component? Another property of Component?

Reply:
We will not care about it. As I mention before, we will just create Component with subcomponent with special name. Let’s say `*`. And this name will be checked in NodeVisitor.